snapcraft/linters/classic_linter.py

```diff
@@ -25,6 +25,8 @@
 
 from .base import Linter, LinterIssue, LinterResult
 
+_HELP_URL = "https://forum.snapcraft.io/t/classic-linter/32228"
```

Comment:
@degville is creating proper documentation links for these. They should be done shortly

Reply:
Updated